pom.xml

```diff
@@ -57,9 +57,29 @@
             <dependency>
                 <groupId>junit</groupId>
                 <artifactId>junit</artifactId>
-                <version>4.12</version>
+                <version>4.11</version>
```

Comment:
I trigged this bug here, thats the reason for the downgrade:

http://stackoverflow.com/a/26222732/102200